pkg/apis/extensions/types.go

```diff
@@ -683,3 +683,30 @@ const (
 	PodSelectorOpExists       PodSelectorOperator = "Exists"
 	PodSelectorOpDoesNotExist PodSelectorOperator = "DoesNotExist"
 )
+
+type DedicatedMachine struct {
+	unversioned.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
+	api.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec defines the desired behavior of this dedicated machine rule.
+	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#spec-and-status
+	Spec DedicatedMachineSpec `json:"spec"`
+}
+
+// DedicatedList is a collection of dedicated machines.
+type DedicatedMachineList struct {
+	unversioned.TypeMeta `json:",inline"`
+	// Standard list metadata.
+	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
+	unversioned.ListMeta `json:"metadata,omitempty"`
+
+	// Items is a list of DedicatedMachines.
+	Items []DedicatedMachine `json:"items"`
+}
+
+type DedicatedMachineSpec struct {
+	// TargetNamespace string `json:"namespace,omitempty"`
```

Comment:
Comment doesn't match field.

Before making an API proposal, please read the documentation about how to do so. Also, it's a good idea to start with a proposal doc, in general, in order to minimize wasted effort.

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/faster_reviews.md
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/coding-conventions.md
https://github.com/kubernetes/kubernetes/blob/master/docs/api.md
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api_changes.md